src/content/capture/video-contenthint/js/main.js

```diff
@@ -0,0 +1,151 @@
+/*
+*  Copyright (c) 2016 The WebRTC project authors. All Rights Reserved.
+*
+*  Use of this source code is governed by a BSD-style license
+*  that can be found in the LICENSE file in the root of the source
+*  tree.
+*/
+
+'use strict';
+
+var srcVideo = document.getElementById('srcVideo');
+var fluidVideo = document.getElementById('fluidVideo');
+var detailedVideo = document.getElementById('detailedVideo');
+
+var srcStream;
+var fluidStream;
+var detailedStream;
+
+var offerOptions = {
+  offerToReceiveAudio: 0,
+  offerToReceiveVideo: 1
+};
+
+function maybeCreateStream() {
+  if (srcStream) {
+    return;
+  }
+  if (srcVideo.captureStream) {
+    srcStream = srcVideo.captureStream();
+    console.log('Captured stream from srcVideo with captureStream',
+      srcStream);
+    call();
+  } else if (srcVideo.mozCaptureStream) {
```

Comment:
For this particular demo, perhaps we don't care about legacy Mozila compatibility. Just handle what's in the standard.